### ISSUE TITLE
Add function to set maximum line width

### ISSUE THREAD

**Issue by mrk-z** [googlefonts/fontra#1041]
It'd be helpful to be able to set a maximum line width for the text box/canvas area, so that the text automatically goes to a new line when it exceeds this threshold.

Currently when we paste a long text string into the text box, we have to separate it into shorter lines manually.

**Comment by justvanrossum** [comment]
Relates to #566. Perhaps this concept makes more sense once we don't need the separate text entry field anymore.

**Comment by justvanrossum** [comment]
The easiest solution would be to add a numeric field to the text entry panel, that specifies the max line length in terms of a "font size" multipier. For example, a value of "100" would mean a maximum width of 100 * 1000 units, for a 1000 units-per-em font. Wrapping should probably be optional (with a checkbox), but on by default.

**Comment by justvanrossum** [comment]
I don't think we should even attempt hyphenations, just break on spaces, and on character if there isn't a space.
